src/articulation.ts

```diff
@@ -298,18 +292,20 @@ export class Articulation extends Modifier {
    * @param type entry in `Vex.Flow.articulationCodes` in `tables.ts` or Glyph code.
    *
    * Notes (by default):
-   * - Glyph codes ending with 'Above' will be positioned ABOVE
-   * - Glyph codes ending with 'Below' will be positioned BELOW
+   * - Even codes will be positioned ABOVE
+   * - Odd codes will be positioned BELOW
    */
   constructor(type: string) {
     super();
 
     this.type = type;
     this.position = ABOVE;
-    this.renderOptions = {
-      fontScale: Tables.NOTATION_FONT_SCALE,
-    };
+    if (!Tables.articulationCodes(this.type)) {
+      if (this.type.charCodeAt(0) % 2 == 0) this.position = ABOVE;
+      else this.position = BELOW;
```

Comment:
Let's make sure we do not need to use `String.codePointAt()` instead:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/codePointAt

charCodeAt doesn't support the entire range of unicode code points.

(I guess passing in numeric code points would make this check a bit easier.... but I do think using the SMuFL glyph directly is probably better.)

Reply:
We should be using codePointAt().

U+1D12B is an alternate codepoint for accidentalDoubleFlat.

0x1D12B is odd.

'\u{1D12B}'.charCodeAt(0) % 2 is 0, which implies it is even. This is incorrect.

'\u{1D12B}'.codePointAt(0) % 2 is 1, which means it is odd. This is correct.

Reply:
done